SpaceShooterGame/SpaceShooterGame/GameScene.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GameScene.swift
+//  SpaceShooterGame
+//
+//  Created by Tristan Garzon on 2018-03-07.
+//  Copyright © 2018 Tristan Garzon. All rights reserved.
+//
+
+
+//Notes: Timer, Enemies, Firing
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    
+    var gbackground:SKEmitterNode!
+    var player:SKSpriteNode!
+    
+    //Score
+    var scoreLabel:SKLabelNode!
+    var score:Int = 0 {
+        didSet {
+            scoreLabel.text = "Score: \(score)"
+        }
+    }
+    
+    override func didMove(to view: SKView) {
+        
+        //Background (File, Position, Time)
+        gbackground = SKEmitterNode(fileNamed: "Background")
```

Comment:
Im not sure this is what you want. and the label of the variable doesnt express much to the intent. I read it as:
"game background is a particle emitter node" it emits background texture.